modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,328 @@
+** Local threshold time unit: Time unit for the Local threshold time unit field
+** Server selection timeout: How long a time to block for server selection before throwing an exception
+** Server selection timeout time unit: Time unit for the Server selection timeout field
+** Socket timeout: Maximum time to wait for a send or receive operation on a socket before the attempt times out
+** Socket timeout unit: Time unit for the  Socket timeout field
+** Min connection pool size: Minimum size for the connection pool
+** Max connection pool size: Maximum size for the connection pool
+** Max wait queue time: Maximum wait queue time for the connection pool
+** Max wait queue time unit: Time unit for the Maximum wait queue time field
+** Max connection life time: Maximum connection lifetime for the connection pool
+** Max connection life time time unit: Time unit for the Maximum connection life time field
+** Max connection idle time: Maximum connection idle time for the connection pool
+** Max connection idle time unit: Time unit for the Maximum connection idle time field
+
+== Changed Operations, Parameters, and Return Types
+
+The following table shows changes to operation names, input parameters, and return types:
+
+[%header%autowidth.spread]
+|===
+|MongoDB 5.x Operation | Changes in MongoDB 6.0
+
+| Count documents a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Query` (JSON)
+
+Return type is now a Long with the count result.
+
+| Create Collection a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Max objects` (Integer)
+ * `Collection size` (Integer)
+ * `Collection size data unit` (TimeUnit)
+
+| Exists collection | Operation is now called Collection exists.
+
+| Create file from payload a|
+
+* Operation is now called Create file.
+* Return type is now a JSON.
+
+
+| Create index a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Field name` (String)
+ * `Sort order` (Enum: ASC or DESC)
+
+| Dump a| Input parameters are now:
+
+* `Output directory` (String)
+* `Output name prefix` (String)
+* `Oplog` (no change)
+* `Operation timeout` (Integer)
+* `Operation timeout unit` (TimeUnit)
+
+Return type is now a List<String> that points to the created files. Each string is a filePath.
+
+| Execute command a|
+
+* Input parameter is `Command` (JSON).
+* Return type is now a JSON that contains the result of the command.
+
+| Find documents a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Query` (JSON)
+ * `Fields` (String)
+ * `Sort by` (JSON)
+ * `Page size` (Integer)
+ * `Limit` (Integer)
+
+This operation supports pagination. Each item returned as a JSON.
+
+| Find files a|
+
+* Input parameters are now:
+ ** `Query` (JSON)
+ ** `Sort` (JSON)
+* Return type is now a JSON list.
+
+| Get file content a|
+
+* Input parameter is now `File id` (JSON).
+* Return type is now a JSON Result object that contains the file content and attributes.
```

Comment:
the return type is a stream with the binary file content. Also it returns the file information as attributes